### PR TITLE
Secp256k1 support

### PR DESCRIPTION
**Pull request by dholms**
Adds secp256k1 support to 
- crypto lib
- did:plc server
- did resolution

### CODE REVIEW

File: packages/crypto/src/did.ts [hunk 1]
```diff
@@ -38,6 +41,8 @@ export const formatDidKey = (jwtAlg: string, keyBytes: Uint8Array): string => {
   }
   if (jwtAlg === 'ES256') {
     keyBytes = p256.compressPubkey(keyBytes)
+  } else if (jwtAlg === 'ES256K') {
+    keyBytes = secp.compressPubkey(keyBytes)
```
Comment (devinivy) [inline packages/crypto/src/did.ts:45]: We could use the new JWT algorithm consts here.

Comment (dholms) [reply packages/crypto/src/did.ts:45]: oh good find thx 🙏

Comment (dholms) [reply packages/crypto/src/did.ts:45]: fixing in the other locations too 👌

**Review by devinivy — APPROVED** [review]
Looks great 👍

**Pull request merged**